src/v2/Components/ArtworkFilter/Utils/getFilterLabelWithCounts.ts

```diff
@@ -0,0 +1,12 @@
+import { themeProps } from "@artsy/palette"
+import { __internal__useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
+
+export const getFilterLabelWithCounts = (
+  filterFacetName: string,
+  filterCounts: number = 0
+) => {
+  const isMobile = __internal__useMatchMedia(themeProps.mediaQueries.xs)
+  const counts = isMobile && filterCounts ? ` • ${filterCounts}` : ""
+
+  return `${filterFacetName}${counts}`
+}
```

Comment:
**thought**: we can implement this like a hook in which we will get the necessary values from the `ArtworkFilterContext` context

```javascript
const useFilterLabelCountByKey = (label: string, key: FilterParamName) => {
  const { selectedFiltersCounts } = useArtworkFilterContext()
  const count = selectedFiltersCounts[key]
  const isMobile = __internal__useMatchMedia(themeProps.mediaQueries.xs)

  return isMobile && count ? ` • ${count}` : ""
}
```

Reply:
Great idea, thanks Dima!